cmake/Config.cmake.in

```diff
@@ -0,0 +1,6 @@
+set(IMGUI_VERSION "@IMGUI_VERSION@")
+
+@PACKAGE_INIT@
+include("${CMAKE_CURRENT_LIST_DIR}/imguiTargets.cmake")
```

Comment:
Since you introduce `TARGETS_EXPORT_NAME` variable in `CMakeLists.txt` logically here should be `include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")`

Reply:
why do you want to bring logic into this :)